### ISSUE TITLE
Could this be used to hide cursor?

### ISSUE THREAD

**Issue by curiousyuvi** [screen-share/element-capture#60]
I want to know if element-capture could be used to hide the mouse cursor from the video stream.

My usecase is to record screen in my chrome extension with cursor hidden.

I don't know much about element-capture, just glanced at the readme.

**Comment by eladalon1983** [comment]
I don't think you need Element Capture for it. I think you need [CursorCaptureConstraint](https://www.w3.org/TR/screen-capture/#dom-cursorcaptureconstraint). IIRC, it's not presently implemented in Chromium; I am not sure about other browsers.

**Comment by curiousyuvi** [comment]
Yeah, that's the problem `CursorCaptureConstraint` is not implemented in Chromium. I thought maybe element-capture could some how help me with that, well nevermind.
